### PR TITLE
Feature/infocard props on site editor

### PR DESCRIPTION
**Pull request by DilipKumar-n**
#### What problem is this solving?

Making 'isFullModeStyle, textMode, textPosition, textAlignment, blockClass' props available in the site Editor for Infocard component.
The reason why I am displaying the above props is that Users/Clients can manage the component by their own from the site editor and there will be minimal/no code level changes required.

#### How to test it?

Link this PR on a workspace and use the block "list-context.info-card-list" in order to test the keyboard module.

#### Screenshots or example usage:

siteEditorprops
siteEditorProps

Opened issue in the Store Discussion Repo
https://github.com/vtex-apps/store-discussion/issues/673

### CODE REVIEW

File: react/components/InfoCard/index.js [hunk 1]
```diff
@@ -25,10 +25,10 @@ import { SanitizedHTML } from '../SanitizedHTML'
...
 const ALLOWED_TAGS = ['p', 'span', 'a', 'div', 'br']
 const ALLOWED_ATTRS = {
-  a: ['class', 'href', 'title', 'target'],
-  span: ['class'],
-  p: ['class'],
-  div: ['class'],
+  a: ['class', 'href', 'title', 'target', 'style'],
+  span: ['class', 'style'],
+  p: ['class', 'style'],
+  div: ['class', 'style'],
```
Comment (victorhmp) [inline react/components/InfoCard/index.js:31]: I don't think we should be adding `style` to the allowed attributes. Just `class` should be enough :)

**Review by victorhmp — CHANGES_REQUESTED** [review]
Please update the CHANGELOG

File: react/components/InfoCard/index.js (hunk 1 above)
Comment (DilipKumar-n) [reply react/components/InfoCard/index.js:31]: @victorhmp The client can use inline styling so no need to have code-level changes. So I have included style tag.

**Comment by DilipKumar-n** [comment]
> Please update the CHANGELOG

Updated the CHANGELOG

File: react/components/InfoCard/index.js (hunk 1 above)
Comment (victorhmp) [reply react/components/InfoCard/index.js:31]: I understood your idea, but we don't want to include that for everyone. Using the inline `style` like that would go against our idea of how developers should be styling their stores. By using just `class`, you can create a CSS class and style it in an actual CSS file, not inline.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - `srcSet` typo in the Image documentation.
 
+## [3.155.9] - 2022-02-03
+
+### Fixed
+- Enable 'isFullModeStyle, textMode, textPosition, textAlignment, blockClass' props available in the site Editor for Infocard component.
```
Comment (victorhmp) [inline CHANGELOG.md:16]: This shouldn't be in this section. Please follow the instructions in our contributing guide: https://github.com/vtex-apps/store-discussion/blob/master/CONTRIBUTING.md.

File: react/components/InfoCard/index.js (hunk 1 above)
Comment (DilipKumar-n) [reply react/components/InfoCard/index.js]: @victorhmp Removed `style` attribute

File: CHANGELOG.md (hunk 2 above)
Comment (DilipKumar-n) [reply CHANGELOG.md]: @victorhmp Updated

File: store/contentSchemas.json [hunk 3]
```diff
@@ -59,6 +75,33 @@
           "description": "admin/editor.info-card.imageActionUrl.description",
           "$ref": "app:vtex.native-types#/definitions/url",
           "default": ""
+        },
+        "textPosition": {
+          "default": "left",
+          "type": "string",
+          "title": "admin/editor.info-card.textPosition.title",
+          "enum": ["left", "center","right"],
```
Comment (igorbrasileiro) [inline store/contentSchemas.json:83]:
```suggestion
          "enum": ["left", "center", "right"],
```

File: store/contentSchemas.json [hunk 4]
```diff
@@ -59,6 +75,33 @@
...
+            "admin/editor.info-card.textPosition.left",
+            "admin/editor.info-card.textPosition.center",
+            "admin/editor.info-card.textPosition.right"
+          ]
+        },
+        "textAlignment": {
+          "default": "left",
+          "type": "string",
+          "title": "admin/editor.info-card.textAlignment.title",
+          "enum": ["left", "center","right"],
```
Comment (igorbrasileiro) [inline store/contentSchemas.json:94]:
```suggestion
          "enum": ["left", "center", "right"],
```

**Review by igorbrasileiro — APPROVED** [review]
Fix those spaces, and LGTM

**Comment by DilipKumar-n** [comment]
> Fix those spaces, and LGTM

Fixed the spaces issue.

**Comment by victorhmp** [comment]
@DilipKumar-n can you provide a workspace with this version linked?

**Comment by DilipKumar-n** [comment]
> @DilipKumar-n can you provide a workspace with this version linked?

Here is the link - https://infocard--rmoils.myvtex.com/

**Review by victorhmp — APPROVED** [review]

**Pull request merged**